File: src/frontend/src/app/models/person.ts
import { PhoneNumber } from "./phone_number";

export class Person
{
    constructor(
        public id:              number,
        public userName:        string,
        public administratorId: number,
        public firstName:       string,
        public lastName:        string,
        public emailAddress:    string,
        public phoneNumber:     PhoneNumber[] 
    ) { }
}

File: src/backend/src/api/Models/Person.cs
namespace Restify.API.Models;

public class Person
{
	public int		Id { get; set; }
	public string	UserName { get; set; }
	public string	FirstName { get; set; }
	public string	LastName { get; set; }
	public string	EmailAddress { get; set; }

	public List<PaymentInformation> PaymentInformations { get; }
	public List<PersonPhoneNumber>	PhoneNumbers { get; }
	public User	User { get; }

	public Person()
	{
		UserName = FirstName = LastName = EmailAddress = string.Empty;
	}
}